packages/react-core/src/next/components/Dropdown/examples/Dropdown.md

```diff
@@ -10,22 +10,36 @@ import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-ico
 
 ## Examples
 
-### Basic
+### Basic dropdowns
+Basic dropdowns present users with a menu of items upon clicking a dropdown toggle. 
+
+A `<Dropdown>` uses a [menu toggle](/components/menu-toggle), which is passed to the `toggle` property, as well as a `<DropdownList>` with multiple `<DropdownItem>` components. To allow a dropdown to change the open state of the menu, pass the `isOpen` property into the `onOpenChange` callback. 
+
+The following example shows a few different states of dropdown items. As shown in this example, dropdown items may: 
+- Use the `isDisabled` property to disable an item.
+- Use the `onClick` property to trigger a callback for an action.
+
+This example also uses a `<Divider>` which passes in `key=”separator”` to split the menu into 2 sections with a horizontal line.
 
 ```ts file="./DropdownBasic.tsx"
 ```
 
 ### With kebab toggle
 
+To style a dropdown menu's toggle with a kebab icon, pass a "plain" `<MenuToggle>` to the `toggle` property. When there isn't enough space for a labeled button, a kebab icon can indicate to users that a dropdown connects to a menu.
```

Comment:
The second sentence feels awkward to me. Maybe just say, "When there isn't enough space for a labeled button, a kebab icon can be used to toggle the dropdown menu open or closed."?

Reply:
agree, I like this change!